src/components/workspace-utils.js

```diff
@@ -504,3 +505,8 @@ export const NoWorkspacesMessage = ({ onClick }) => {
     ])
   ])
 }
+
+//TODO: Workspace should contain it's cloud provider in a better fashion.
+export const getCloudProviderFromWorkspace = workspace => {
+  return workspace.azureContext ? cloudProviderTypes.AZURE : cloudProviderTypes.GCP
```

Comment:
You should be able to use `cloudPlatform`.

![image](https://user-images.githubusercontent.com/484484/203083254-5251b695-5e79-4cf7-808e-8eaafdee90bf.png)

Reply:
Thank you - that's perfect! I'll get on making the adjustments.

Reply:
It seems like this field isn't available yet in dev

Reply:
You may need to change the code to use it request it when fetching workspace information (it's not particularly new). We are using it [here](https://github.com/DataBiosphere/terra-ui/blob/ae79dba284c83de2cc1d57300c44742b0bd3636f/src/pages/workspaces/List.js#L358).

Reply:
@n8landolt did you look into this?

Reply:
I didn't have all that much time yesterday to look deeply, but a quick search didn't net me what I wanted.
I'm going to see if I can do it now.

Reply:
The getWorkspaceDetails endpoint doesn't return the cloudPlatform, even when specifically requested.

Reply:
It looks like it just wasn't implemented-- I created https://broadworkbench.atlassian.net/browse/WOR-646.

Reply:
Thank you! This will be very helpful.